buildkite/bazel-central-registry/bcr_presubmit.py

```diff
@@ -62,32 +60,20 @@ def error(msg):
 
 def get_target_modules():
     """
-    If the `MODULE_NAME` and `MODULE_VERSION(S)` are specified, calculate the target modules from those env vars.
-    Otherwise, calculate target modules based on changed files from the main branch.
+    Calculate target modules based on changed files from the main branch.
     """
-    modules = []
-    if "MODULE_NAME" in os.environ:
-        name = os.environ["MODULE_NAME"]
-        if "MODULE_VERSION" in os.environ:
-            modules.append((name, os.environ["MODULE_VERSION"]))
-        elif "MODULE_VERSIONS" in os.environ:
-            for version in os.environ["MODULE_VERSIONS"].split(","):
-                modules.append((name, version))
-
-    if modules:
-        return list(set(modules))
-
     # Get the list of changed files compared to the main branch
     output = subprocess.check_output(
         ["git", "diff", "main...HEAD", "--name-only", "--pretty=format:"]
     )
+    modules = []
```

Comment:
Why is it a list in the first place? You can make modules a set and remove "set" from 76

Reply:
Ah, good point, fixed!